src/lib/commandCenter/panels/ai.svelte

```diff
@@ -0,0 +1,227 @@
+                        {/each}
+                    {/if}
+                </div>
+            </div>
+        </div>
+    {/if}
+
+    <div class="footer" slot="footer">
+        <div class="u-flex u-cross-center u-gap-4">
+            <AvatarInitials size={32} name={$user.name} />
+            <form
+                class="input-text-wrapper u-width-full-line"
+                style="--amount-of-buttons: 1;"
+                on:submit|preventDefault={handleSubmit}>
+                <!--  svelte-ignore a11y-autofocus -->
+                <input
+                    type="text"
+                    class="input-text"
+                    placeholder="Ask a question..."
+                    autofocus
+                    bind:value={$input}
+                    disabled={$isLoading} />
+                <div class="options-list">
+                    <button
+                        class="options-list-button"
+                        aria-label="ask AI"
+                        type="submit"
+                        disabled={!$input.trim() || $isLoading}>
+                        <span class="icon-arrow-sm-right" aria-hidden="true" />
+                    </button>
+                </div>
+            </form>
+        </div>
+
+        <div class="u-flex u-main-end u-cross-center u-gap-16 u-margin-block-start-16">
+            <div class="u-flex u-cross-center u-gap-4">
+                <kbd class="kbd">Enter</kbd>
+                <span>to search</span>
+            </div>
+            <div class="sep" />
+            <div class="u-flex u-cross-center u-gap-4">
+                <kbd class="kbd">Esc</kbd>
+                <span>to {$subPanels.length === 1 ? 'close' : 'go back'}</span>
+            </div>
+        </div>
+    </div>
+</Template>
+
+<style lang="scss">
+    .content {
+        overflow-y: auto;
+        padding: 1rem;
+
+        .logo {
+            display: flex;
+            width: 1.5rem;
+            height: 1.5rem;
+            justify-content: center;
+            align-items: center;
+            flex-shrink: 0;
+
+            border-radius: 0.25rem;
+            background: #282a3b;
+        }
+
+        .answer {
+            overflow: hidden;
+
+            p {
+                white-space: pre-wrap;
+            }
+        }
+
+        h2 {
+            color: hsl(var(--color-neutral-70));
+        }
+
+        .examples {
+            display: flex;
+            flex-direction: column;
+
+            li {
+                padding: 0.59375rem 0.5rem;
+
+                button {
+                    &:hover {
+                        opacity: 0.75;
+                    }
+
+                    i {
+                        color: hsl(var(--color-neutral-70));
+                    }
+                }
+            }
+        }
+    }
+
+    .footer {
+        .sep {
+            width: 1px;
+            height: 1.5rem;
+            background-color: hsl(var(--color-neutral-150));
+        }
+    }
```

Comment:
Designers said it wasn't needed, and I think it makes sense as well, some things are going to be console specific rather than a design system component.